app/views/consultation/sections/_outcome_details.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render "shared/attachments_list",
```

Comment:
Again, I like this file but I'm not in love with the name/location.

Reply:
Thanks Keith! That's updated now in commit [Update consultation and cfe to use outcome_details shared partial](https://github.com/alphagov/frontend/pull/5074/commits/15f35fd3b915d8230fcd1eca97b3e3bf3173fa9c)